activitystatusimg.php

```diff
@@ -1,61 +1,57 @@
 <?php
-
 define('NO_DEBUG_DISPLAY', true);
-
 require(dirname(__FILE__).'/../../config.php');
 require_once($CFG->dirroot.'/lib/completionlib.php');
 
 $idnumber = required_param('idnumber', PARAM_TEXT);
+$iconset = optional_param('iconset', '', PARAM_TEXT);
```

Comment:
does this need to be param_text? - can it be PARAM_ALPHA or PARAM_INT?

Reply:
done